### PR TITLE
fix(importsTests)

### PR DESCRIPTION
**Pull request by LironHazan**

### CODE REVIEW

File: spec/imports/imports.spec.js [hunk 1]
```diff
@@ -16,97 +16,52 @@ describe('imports section', function() {
...
 
             components.layout.saveBlueprint(); //click save blueprint button
-            browser.refresh();
-
-            expect(components.layout.countElements(element.all(by.repeater('imports in globals.stencils.importsData')))).toBe(10);
-
             browser.sleep(200).then(done);
 
+            browser.driver.navigate().refresh();
+            expect(components.layout.countElements(element.all(by.repeater('imports in globals.stencils.importsData')))).toBe(2);
```
Comment (Kinnza) [inline spec/imports/imports.spec.js]: Why do we need to refresh? doesnt it happen update the table? why isnt wait good enough?

File: spec/imports/imports.spec.js [hunk 2]
```diff
@@ -115,12 +70,37 @@ describe('imports section', function() {
...
+        it('node types should exist', function(done) {
+            expect(components.layout.countElements(element.all(by.repeater('nodeType in globals.stencils.nodeTypes')))).toBe(5);//default amount of node types
+            browser.sleep(200).then(done);
+        });
+        it('should orderBy type', function(done) {
+            components.imports.page.orderByNodeType();
+            expect(components.imports.page.checkFirstNodeType()).toBe('cloudify.chef.nodes.WebServer');//last node type
 
-            expect(components.layout.countElements(element.all(by.repeater('imports in globals.stencils.importsData')))).toBe(9);
+            browser.sleep(200).then(done);
```
Comment (Kinnza) [inline spec/imports/imports.spec.js]: Why are we sleeping after the test?

File: spec/imports/imports.spec.js (hunk 1 above)
Comment (LironHazan) [inline spec/imports/imports.spec.js]: the refresh is done after the save, yes - save will fetch and display the updated data but the one who wrote the test probably used refresh for the other scenario - when adding/removing import and not saving the blueprint - in that case refreshing will cause the data to disappear cause the blueprint wasn't saved.

File: spec/imports/imports.spec.js (hunk 2 above)
Comment (LironHazan) [inline spec/imports/imports.spec.js]: we sleep after every test and pass the done to the promise.

**Pull request merged**